src/drt/src/io/io.cpp

```diff
@@ -176,7 +156,7 @@ void io::Parser::setObstructions(odb::dbBlock* block)
       continue;
     }
     frLayerNum layerNum = tech_->name2layer[layerName]->getLayerNum();
-    auto blkIn = make_unique<frBlockage>();
+    auto blkIn = make_unique<frBlockage>(blockage->getBBox());
```

Comment:
Does it make more sense to store the `dbObstruction` than the bbox of the obstruction?